### ISSUE TITLE
OSNotification payload.body is crashing when no title

### ISSUE THREAD

**Issue by allaire** [OneSignal/OneSignal-iOS-SDK#117]
When I send a notification from the OneSignal New Message interface, and I don't specify a Title, `print(notification.payload.body)` is crashing when I receive the notification (at least in Swift).

When I specify a title, no crash occurs.

In both cases, an english content (body) was specified (it's required anyway)

**Comment by allaire** [comment]
@jkasten2 I'll be around all day if you need any more help debugging/reproducing 👍

**Comment by jkasten2** [comment]
@allaire Is this happening in the received or open callback? could you add the code around it for context?

Are you seeing this issue on the 2.2.0 SDK? Also include the XCode version and iOS version you are testing on.

Thanks.

**Comment by allaire** [comment]
@jkasten2 

Received callback.

I'm still running on Xcode 7 with Swift 2, currently migrating to Swift 3.

Happens on 2.2.0 sdk.

**Comment by jkasten2** [comment]
@allaire I just tested our 2.2.0 SDK on a Xcode 7.3.1 swift project and included you print line without any issues.

Can you confirm the crash is coming from that line by attaching the full stack trace of the crash?

**Comment by allaire** [comment]
@jkasten2 Did you test by sending a test notification from the panel with no title?

Here's my init:

```
        // Push Notifications
        let oneSignalSettings: [AnyHashable: Any] = [kOSSettingsKeyAutoPrompt: false, kOSSettingsKeyInFocusDisplayOption: OSNotificationDisplayType.none.rawValue]
        
        if Router.environment() == "production" {
            OneSignal.initWithLaunchOptions(launchOptions, appId: Constant.Vendor.OneSignalProdAppId, handleNotificationReceived: { (notification) in
                self.didReceiveRemoteNotification(notification!)
            }, handleNotificationAction: { (result) in
                self.didReceiveRemoteNotification((result?.notification)!)
            }, settings: oneSignalSettings)
        } else {
            OneSignal.initWithLaunchOptions(launchOptions, appId: Constant.Vendor.OneSignalDevAppId, handleNotificationReceived: { (notification) in
                self.didReceiveRemoteNotification(notification!)
            }, handleNotificationAction: { (result) in
                self.didReceiveRemoteNotification((result?.notification)!)
            }, settings: oneSignalSettings)
            OneSignal.setLogLevel(.LL_DEBUG, visualLevel: .LL_ERROR)
        }
```

**Comment by jkasten2** [comment]
I did, I left all the other settings default. Could you share your notification id and link this issue in? We can check the exact notification on our end.

Thanks for the code, could in include your `didReceiveRemoteNotification` function as well?

**Comment by allaire** [comment]
@jkasten2 I followed-up via Intercom in our account.

Thanks

**Comment by jkasten2** [comment]
@allaire I have fixed the `nil` body issue in commit b569c781711704ff8deb04690ea11c7c828a38e6. It will be included in the next release.
